saritasa_invocations/db_k8s.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+import invoke
+
+from . import _config, k8s
+
+
+@invoke.task
+def create_dump(
+    context: invoke.Context,
+    dbname: str,
+    host: str,
+    port: str,
+    username: str,
+    password: str,
+    file: str = "",
+    additional_params: str = "",
+) -> None:
+    """Execute dump command in db pod."""
+    config = _config.Config.from_context(context)
+    command = _generate_dump_command(
+        context,
+        file=file,
+        dbname=dbname,
+        host=host,
+        port=port,
+        username=username,
+        additional_params=additional_params,
+    )
+    k8s.success(context, f"Entering into db with {command}")
+    db_exec_command = _generate_exec_command(context)
+    context.run(
+        f"{db_exec_command} -- {command}",
+        watchers=(
+            invoke.Responder(
+                pattern=config.db.password_pattern,
+                response=f"{password}\n",
+            ),
+        ),
+    )
+
+
+@invoke.task
+def get_dump(
+    context: invoke.Context,
+    file: str = "",
+) -> str:
+    """Download db data from db pod if it present."""
+    config = k8s.get_current_env_config_from_context(context).db_config
+    file = _get_db_k8s_dump_filename(context, file)
+    file_to_download = f"{file}.sql"
```

Comment:
Implicit adding of `.sql` extension may be confusing. For example, user generated dump using
`inv db-k8s.create-dump --file=fname.dump.sql`.
If user will try to download it using
`inv db-k8s.get-dump --file=fname.dump.sql`,
user will get an error about not existing file